bank.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'account'
+# SauceBank
+class Bank
+  attr_accessor :accounts
+  def initialize
+    @accounts ||= load_accounts
+  end
+
+  def load_accounts
```

Comment:
This doesn't feel like it needs to be pulled out into its own method.  What benefit do we get from this?

Reply:
This method will eventually load accounts from CSV, and i wanted to ensure I was using Memoization for that.